packages/eslint-plugin/rules/__tests__/no-store-string-literals.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import { RuleTester } from 'eslint';
+
+/**
+ * Internal dependencies
+ */
+import rule from '../no-store-string-literals';
+
+const ruleTester = new RuleTester( {
```

Comment:
This rule tester from ESLint is amazing. I love how easy it is to create all the checks. Great work here 👏🏻